nodejs/dist/uws.js

```diff
@@ -94,14 +94,22 @@ class Socket {
      *
      * @param {String|Buffer} message The message to send
      * @param {Object} options Send options
+     * @param {Function} cb optional callback
      * @public
      */
-    send(message, options) {
+    send(message, options, cb) {
         /* ignore sends on closed sockets */
-        if (!this.nativeSocket) return;
+        if (typeof options === 'function') {
+            cb = options;
+            options = {};
```

Comment:
Maybe we could make this `options = null` instead. I'm pretty sure {} creates a new object which is not neccesary. We only want `options` to evaluate to `false`.